packages/core/src/http/requestBuilder.ts

```diff
@@ -111,6 +114,7 @@ export interface RequestBuilder<BaseUrlParamType, AuthParams> {
     parameters?: Record<string, unknown> | null,
     prefixFormat?: ArrayPrefixFunction
   ): void;
+  template(name: string, value: PathTemplateTypes): void;
```

Comment:
Please explain the addition of this new template param function, does it also require changes in the SDK?

Have you ensured that `appendPath` functionality remains the same with this change?